commit 930ba4a374b96560ef9fde2145cdc454a164ddcc
Author: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date:   Wed Nov 27 09:32:49 2013 -0800

    Staging: tidspbridge: disable driver
    
    There seems to be no active maintainer for the driver, and there is an
    unfixed security bug, so disable the driver for now.
    
    Hopefully someone steps up to be the maintainer, and works to get this
    out of staging, otherwise it will be deleted soon.
    
    Reported-by: Nico Golde <nico@ngolde.de>
    Reported-by: Dan Carpenter <dan.carpenter@oracle.com>
    Cc: Omar Ramirez Luna <omar.ramirez@copitl.com>
    Cc: Omar Ramirez Luna <omar.ramirez@ti.com>
    Cc: Kanigeri, Hari <h-kanigeri2@ti.com>
    Cc: Ameya Palande <ameya.palande@nokia.com>
    Cc: Guzman Lugo, Fernando <fernando.lugo@ti.com>
    Cc: Hebbar, Shivananda <x0hebbar@ti.com>
    Cc: Ramos Falcon, Ernesto <ernesto@ti.com>
    Cc: Felipe Contreras <felipe.contreras@gmail.com>
    Cc: Anna, Suman <s-anna@ti.com>
    Cc: Gupta, Ramesh <grgupta@ti.com>
    Cc: Gomez Castellanos, Ivan <ivan.gomez@ti.com>
    Cc: Andy Shevchenko <ext-andriy.shevchenko@nokia.com>
    Cc: Armando Uribe De Leon <x0095078@ti.com>
    Cc: Deepak Chitriki <deepak.chitriki@ti.com>
    Cc: Menon, Nishanth <nm@ti.com>
    Cc: Phil Carmody <ext-phil.2.carmody@nokia.com>
    Cc: Ohad Ben-Cohen <ohad@wizery.com>
    Cc: stable <stable@vger.kernel.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/staging/tidspbridge/Kconfig b/drivers/staging/tidspbridge/Kconfig
index 165b918b8171..1b6d581c438b 100644
--- a/drivers/staging/tidspbridge/Kconfig
+++ b/drivers/staging/tidspbridge/Kconfig
@@ -4,7 +4,7 @@
 
 menuconfig TIDSPBRIDGE
 	tristate "DSP Bridge driver"
-	depends on ARCH_OMAP3 && !ARCH_MULTIPLATFORM
+	depends on ARCH_OMAP3 && !ARCH_MULTIPLATFORM && BROKEN
 	select MAILBOX
 	select OMAP2PLUS_MBOX
 	help